spec/features/edit_user_story_spec.rb

```diff
@@ -17,6 +17,15 @@
     expect(page).to have_content 'As a user I want beef to eat'
   end
 
+  it "fails if someone else edits in the meantime" do
+    error = ActiveRecord::StaleObjectError.new(@user_story, :update)
+    allow_any_instance_of(UserStory).to receive(:update_attributes).and_raise(error)
+    visit "/projects/#{@project.id}/user_stories/#{@user_story.id}/edit"
+    fill_in 'user_story_definition', :with => 'As a user I want beef to eat'
+    click_button 'Save and Close'
+    expect(page).to have_content 'Another person has just updated that record'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.